change/@fluentui-react-components-102726a3-c1d7-4c19-9454-412a1a1d8b7e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: exports DialogContent as unstable",
```

```suggestion
  "type": "patch",
  "comment": "feat: exports DialogContent as unstable",
```

"none" will not trigger a release.